src/provider.ts

```diff
@@ -110,7 +110,7 @@ export class AnkrProvider {
      * @param params A GetNftTransfersRequest object.
      * @returns Promise<GetNftTransfersReply>
      */
-    async getNftTransfers(params: GetTransfersRequest): Promise<GetNftTransfersReply> {
+    async getNftTransfers(params: GetNftTransfersRequest): Promise<GetNftTransfersReply> {
```

Comment:
`GetTransfersRequest` is used for the `getTokenTransfers` and for the `getNftTransfers`. Therefore, the interface has a name without specifics. Let's change the `@params` description

Reply:
@memekas wdyt on GetTransfersRequest as main type + 2 type aliases like GetTokenTransfersRequest and GetNftTransfersRequest ?
Like
```
export interface GetTokenTransfersRequest extends GetTransfersRequest {}
export interface GetNftTransfersRequest extends GetTransfersRequest {}
```